SimplenoteScreenshots/SimplenoteScreenshots.swift

```diff
@@ -204,11 +254,44 @@ class SimplenoteScreenshots: XCTestCase {
 
         snapshot("\(title)-\(mode)")
     }
+
+    let noteForDetailScreenshot = "Lemon Cake & Blueberry"
+    let noteForInterlinkingScreenshot = "Colors"
 }
 
 extension XCUIElement {
 
-    func pasteText(text: String) -> Void {
+    /// Workaround to type text into secure text fields due to the different behaviors across
+    /// Simulators.
+    func typeSecureText(_ text: String, using app: XCUIApplication) -> Void {
+        tap()
+
+        // At the time of writing, typing in a secure text field didn't work in some of the
+        // Simulators on which we want to take screenshots. On top of that, the workaround to type
+        // in the secure text field doesn't work in some other Simulators. Therefore, we need to
+        // know which approach to use at runtime.
+        if requiresSecureTextFieldWorkaround(using: app) {
+            paste(text: text)
+        } else {
+            typeText(text)
+        }
+    }
+
+    private func requiresSecureTextFieldWorkaround(using app: XCUIApplication) -> Bool {
+        // At the time of writing, these tests run on the following iOS 14.3 Simulators as defined
+        // in the Fastfile.
+        //
+        // - iPhone 8 Plus
+        // - iPhone Xs
+        // - iPad Pro 12.9" 2nd generation
+        // - iPad Pro 12.9" 3rd generation
+        //
+        // Of those, the only one requiring the secure text field workaround is the iPhone 8 Plus
+        // one.
+        return app.isDeviceIPhone8Plus()
+    }
```

Comment:
This is not true anymore, because we now run on iPhone 8 instead of 8 Plus and X instead of Xs (Max).

I think it's worth keeping this code in here even if technically unused, just in case we'll have problems with the secure text field in the future, a component which has proven to be fiddly over the years...